packages/localizations/src/utils/generate.ts

```diff
@@ -164,6 +165,7 @@ run(beBY, 'be-BY');
 run(csCZ, 'cs-CZ');
 run(daDK, 'da-DK');
 run(deDE, 'de-DE');
+run(enGB, 'en-GB');
```

Comment:
Auto-generation will not work as expected if the file has custom logic inside it. 

Let's drop the changes of `generate.ts` file.